### PR TITLE
Admin table refactor and create admin

### PR DESCRIPTION
**Pull request by manishacharya60**
**Frontend:**

Added popup design for new admin button.

**Backend:**

Implemented the popup window to create new admin into the database collection

### CODE REVIEW

File: components/Forms/CreateAdminPopupWindow.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      headers: {
+        // Specify the content type in the headers
+        'Content-Type': 'application/json',
+      },
+      // Stringify the JSON body
+      body: data,
+    });
+
+    setShowPopup(false);
+  };
```
Comment (JiashuHarryHuang) [inline components/Forms/CreateAdminPopupWindow.tsx]: After creating the admin, should show a message that alarms the user about it, either success or failure. You can use antd [message](https://ant.design/components/message)

File: pages/api/admin/index.ts [hunk 2]
```diff
@@ -24,8 +25,59 @@ export default async function handler(
...
+        // Check if the user already exists in database
+        const checkExisting = (await Admins.findOne({
+          email,
+        })) as QueriedAdminData;
+
+        // If the user already exists, return an error
+        if (checkExisting) {
+          res.status(422).json({ message: 'Admin email already exists' });
+          throw new Error('Admin email already exists');
+        }
```
Comment (JiashuHarryHuang) [inline pages/api/admin/index.ts:57]: Before you insert the user, can you first check if the current user has an admin status?

**Review by JiashuHarryHuang — CHANGES_REQUESTED** [review]

**Comment by manishacharya60** [comment]
Implemented the message feature

**Review by JiashuHarryHuang — APPROVED** [review]
LGTM!

**Pull request merged**